packages/fetchai/connections/http_client/connection.yaml

```diff
@@ -11,7 +11,7 @@ fingerprint:
 fingerprint_ignore_patterns: []
 protocols:
 - fetchai/http:0.1.0
-class_name: HTTPConnection
+class_name: HTTPClientConnection
```

Comment:
Minor mistake in the htto_client connection.

Reply:
unfortunately, this means we have to bump the package to 0.2.0, so this needs to be changed as well as all the references to it.

Reply:
Do you want me to do this as part of this PR?